src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.db;
+
+import core.basesyntax.enums.Operation;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private Map<String, Integer> fruitQuantities = new HashMap<>();
+    private Map<Operation, OperationStrategy> strategies;
```

Comment:
Strategies placed in storage not good idea, it`s different things

Reply:
refactored